src/elexmodel/client.py

```diff
@@ -229,6 +235,10 @@ def get_estimates(
             handle_unreporting=handle_unreporting,
         )
 
+        if new_estimands:
```

Comment:
Since the `new_estimands` default is an empty dictionary, this means that if I don't specify a `new_estimands` the Estimandizer never gets called, right? but what if I want the estimand to be something for which we have a pre-defined function like party vote share?

Reply:
Yes, if you don't specify a new estimand it will never get called.

If you want to call a predefined function, for instance "party_vote_share" you just need to pass in {"party_vote_share": None} since the generate function checks that if the value is None, then the estimand must be a pre-written function. If you make a new function and want to pass that as well, you can do {"party_vote_share": None, "new_function": [pass in new function here]} and then it will run that function.